perl/CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+Please read [CONTRIBUTING](https://github.com/cucumber/gherkin/blob/master/CONTRIBUTING.md) first.
+You should clone the [cucumber/gherkin](https://github.com/cucumber/gherkin) repo if you want
+to contribute.
+
+You will need `cpanm` installed on your system
+
+## Run tests
+
+### Using make
+
+Just run `make` from this directory.
+
+## Make a release
+
+    # Change version in `dist.ini` and run
+    make release
```

Comment:
Should be able to automate upload too very easily, to save people having to log in to PAUSE.